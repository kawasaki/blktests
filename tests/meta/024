#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2025 Western Digital Corporation or its affiliates.
#
# Test skipping from test_device_array()

. tests/meta/rc

DESCRIPTION="skip in test_device_array()"

test_device_array() {
	SKIP_REASONS+=("(╯°□°)╯︵ ┻━┻")
}
