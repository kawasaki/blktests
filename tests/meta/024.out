Running meta/024
Test complete
